src/main/java/frc/robot/subsystems/swerve/ModuleIOSim.java

```diff
@@ -67,4 +75,18 @@ public void setTurnVoltage(double volts) {
     turnAppliedVolts = MathUtil.clamp(volts, -12.0, 12.0);
     turnSim.setInputVoltage(turnAppliedVolts);
   }
+
+  @Override
+  public void setDriveSetpoint(double metersPerSecond) {
+    setDriveVoltage(
+        driveFeedforward.calculate(metersPerSecond)
+            + driveController.calculate(
+                driveSim.getAngularVelocityRadPerSec() * Module.WHEEL_RADIUS, metersPerSecond));
```

Comment:
consider formatting this better? maybe not indenting more and more as it is harder to read?

Reply:
swapping the order of the feedforward and pid gives this
```
@Override
  public void setDriveSetpoint(double metersPerSecond) {
    setDriveVoltage(
        driveController.calculate(
                driveSim.getAngularVelocityRadPerSec() * Module.WHEEL_RADIUS, metersPerSecond)
            + driveFeedforward.calculate(metersPerSecond));
  }
```
does this look better to you? other than that im not sure how to format it better, its just an awkward set of method calls.